pywinauto/unittests/test_mouse.py

```diff
@@ -1,16 +1,27 @@
 """Tests for mouse.py"""
 
 import time
+import copy
 import sys
 import os
-import win32clipboard
 import unittest
-
-sys.path.append(".")
-from pywinauto.application import Application
-from pywinauto.keyboard import SendKeys
-from pywinauto import mouse
-from pywinauto.timings import Timings
+if sys.platform == 'win32':
+    import win32clipboard
+    sys.path.append(".")
+    from pywinauto.application import Application
+    from pywinauto.keyboard import SendKeys
+    from pywinauto import mouse
+    from pywinauto.timings import Timings
+else:
+    import subprocess
+    from Xlib.display import Display
+    parent_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    sys.path.insert(0, parent_dir)
```

Comment:
Is there some special case to use `.insert(0, ...)` instead of just `.append(...)`?